install.json

```diff
@@ -13,19 +14,35 @@
   "options": {
     "properties": {
       "piwikdomain": {
-        "title": "The URL of your Piwik installation",
+        "title": "The URL of your Matomo installation",
         "type": "string",
         "format": "url",
-        "placeholder": "https://piwik.example.com",
-        "description": "Enter the URL where your Piwik is installed, eg https://piwik.example.com",
-        "required": true
+        "placeholder": "https://matomo.example.com",
+        "description": "Enter the URL where your Matomo is installed, eg https://matomo.example.com",
+        "required": true,
+        "order": 1
       },
       "piwikidsite": {
-        "title": "Piwik Site ID",
-        "description": "Enter the ID of the site you want to track the data into. You can retrieve the ID of your site by logging into your Piwik and going to 'Administration => Websites: Manage'.",
+        "title": "Matomo Site ID",
+        "description": "Enter the ID of the site you want to track the data into. You can retrieve the ID of your site by logging into your Matomo and going to 'Administration => Websites: Manage'.",
         "type": "integer",
         "minimum": 1,
-        "required": true
+        "required": true,
+        "order": 2
+      },
+      "disableCookies": {
+        "title": "Disable all tracking cookies",
+        "description": "Disables all first party cookies",
+        "type": "boolean",
+        "default": false,
+        "order": 3
+      },
+      "clientSideDNT": {
+        "title": "Enable client side DoNotTrack detection",
+        "description": "So tracking requests will not be sent if visitors do not wish to be tracked."
```

Comment:
Trailing comma `,` missing. And I checked back and forth the blow `:` why it is shown red in GitHub viewer 😄.

Reply:
you are totally right. I wondered about it too, but then forgot about it. And my editor complained about it, but it was just outside the view.